WalletWasabi.Gui/CommandLine/PasswordFinder.cs

```diff
@@ -37,7 +37,7 @@ internal static void Find(string secret, string language, bool useNumbers, bool
 
 			Console.Write("Enter a likely password: ");
 
-			var password = PasswordConsole.ReadPassword();
+			var password = Console.ReadLine();
```

Comment:
????

Reply:
I try to guess what you want.
> for example the password must be written in cleartext it doesnt get masked

Reply:
No, I meant that's the bug. Anyway, it turns out this bug was caused by the attachtoparentconsole and the password was never even accepted so the daemon never worked on windows only for commands those didn't require inputs.